hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -120,6 +120,9 @@
   @Parameter(names = {"--spark-schema-length-threshold"}, description = "The maximum length allowed in a single cell when storing additional schema information in Hive's metastore.")
   public int sparkSchemaLengthThreshold = 4000;
 
+  @Parameter(names = {"--hive-version"}, description = "hive version that hudi use, not suggests user to specific", required = false)
```

Comment:
can you help me understand why would someone wants to override to use a diff hive version than whats actually being used?

Reply:
@nsivabalan I also think this is confusing to allow user overwriting the version, as we always retrieve the version info.